src/evm/WormholeVerifier.sol

```diff
@@ -1391,19 +1422,21 @@ contract WormholeVerifier is EIP712Encoding {
     assembly ("memory-safe") { sstore(schnorrDataSlot, newEntry) }
   }
 
-  function _setSchnorrShardId(uint32 keyIndex, uint8 signerIndex, bytes32 newSchnorrId) internal {
+  function _setSchnorrShardId(uint32 keyIndex, uint8 signerIndex, bytes32 pubKeyX, bytes32 pubKeyY) internal {
     uint256 shardIdSlot = _slotShardMapId(keyIndex, signerIndex);
     assembly ("memory-safe") {
-      sstore(shardIdSlot, newSchnorrId)
+      sstore(sub(shardIdSlot, 1), pubKeyX)
+      sstore(shardIdSlot, pubKeyY)
     }
   }
 
   function _slotShardMapShard(uint32 keyIndex, uint8 signerIndex) internal pure returns (uint256) {
     return SLOT_SCHNORR_SHARD_MAP_SHARD | (keyIndex << 8) | signerIndex;
   }
 
+  // Each ShardId take 2 slots, shardIdSlot -1 and shardIdSlot
```

Comment:
I'm not understanding why we would take `shardIdSlot-1` and `shardIdSlot` instead of `shardIdSlot` and `shardIdSlot+1`. It seems to me that `shardIdSlot-1` would overlap with the space reserved for the previous struct when `keyIndex=0` and `signerIndex=0`.